remediation_worker/jobs/azure_security_center_enable_ddos_protection/constraints.txt

```diff
@@ -0,0 +1,25 @@
+adal==1.2.5
```

Comment:
We are pinning the package version by specifying the hash for the packages.

You can make use of the a hash generating took like [pip-tool ](https://github.com/jazzband/pip-tools) to generate them.